pinecone/manage.py

```diff
@@ -49,7 +50,7 @@ def __str__(self):
 
 
 def _get_api_instance():
-    client_config = Config.OPENAPI_CONFIG
+    client_config = copy.deepcopy(Config.OPENAPI_CONFIG)
```

Comment:
This is what we're doing in `index.py`, I think it's probably prudent to do that here as well rather than possibly changing fields on the singleton.

https://github.com/pinecone-io/pinecone-python-client/blob/7ff4ed877ac3e97eb557b747a072a84e57764b91/pinecone/index.py#L92